internal/ignition/ignition.go

```diff
@@ -1234,16 +1237,17 @@ func GetServiceIPHostnames(serviceIPs string) string {
 	return content
 }
 
-func (g *installerGenerator) runCreateCommand(installerPath, command string, envVars []string) error {
+func (g *installerGenerator) runCreateCommand(ctx context.Context, installerPath, command string, envVars []string) error {
+	log := logutil.FromContext(ctx, g.log)
 	cmd := exec.Command(installerPath, "create", command, "--dir", g.workDir)
 	var out bytes.Buffer
 	cmd.Stdout = &out
 	cmd.Stderr = &out
 	cmd.Env = envVars
 	err := cmd.Run()
 	if err != nil {
-		g.log.Errorf("error running openshift-install create %s", command)
-		g.log.Error(out.String())
+		log.WithError(err).
+			Errorf("error running openshift-install create %s, stdount: %s", command, out.String())
```

Comment:
```suggestion
			Errorf("error running openshift-install create %s, stdout: %s", command, out.String())
```

Reply:
The returned error is wrapped as well with the first 140 chars